polling_stations/templates/fragments/you_dont_need_poll_card.html

```diff
@@ -1,16 +1,27 @@
 {% load i18n %}
-{% if voter_id_pilot %}
```

Comment:
Since we're deleting this, does it make sense to also delete the method in ee_wrapper, and where it's added to  `context` in `data_finder/views.py`?

Reply:
The context assignment has been removed now 🙂

Reply:
Do you know if we're still expecting NI voter id information to be in the metadata? Looking at https://elections.democracyclub.org.uk/api/elections.json?postcode=BT15%203JX&future=1&current=1%27 metadata is `null` for all ballots. So if we know EE isn't using the `metadata` field to communicate about voter id - and we're going to use the `requires_voter_id` key, then I think we should drop that logic too. If it is the case that the information might come by either route (and that wouldn't be an error) then I can see we should leave it in.

Reply:
I confirmed in tech surgery that we're not expecting voter id info for Northern Ireland in the `metadata` object, so I think all that can be removed as part of this work:

Reply:
Grand, I assume that same confirmation also applies to the 2019 voter id trials so I shall remove it